cosmo/netboxclient.py

```diff
@@ -375,7 +375,7 @@ def get_data(self, device_config):
         r = self.query(query)
         graphql_data = r['data']
 
-        static_routes = self.query_rest("/api/plugins/routing/staticroutes/", {"device": device_list})
+        static_routes = self.query_rest("api/plugins/routing/staticroutes/", {"device": device_list})
```

Comment:
why do we have this change?

Reply:
`query_rest` already appends a `/` to the base URL.
We debugged stuff and wondered, if the double slash was a problem - normally its not - and it was not the problem.

Reply:
yea no its not. I think we should use more of urllib in the future (maybe this is something that can be adressed for future refactor?)